CONTRIBUTING.md

```diff
@@ -0,0 +1,34 @@
+# Contribution guidelines
+
+The Wikimate projects welcomes contributions of [all kinds](https://allcontributors.org/docs/en/emoji-key),
+no matter how small!
+Feel free to open [an issue](https://github.com/hamstar/Wikimate/issues/new)
+to discuss your suggestion or request,
+or submit your changes directly as a
+[pull request](https://docs.github.com/en/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork)
+if you prefer.
+
+Below are some guidelines that might help you prepare your contribution.
+
+1. **Limit pull requests limited to a single feature or bugfix.**
+   Unrelated changes should be sent as separate PRs.
+   The exception are minor cleanup changes,
+   which can be included (as a separate commit) in the PR that prompted them.
+2. **Use atomic commits**.
+   Try to create commits that are as small as possible
+   while still representing a self-contained set of changes,
+   and use descriptive commit messages.
+   We recommend [these principles](https://chris.beams.io/posts/git-commit/#seven-rules)
+   for writing great commit messages (but they're not enforced, so don't sweat it 🙂).
+3. **Include a `CHANGELOG.md` entry in every pull request**.
+   This makes it much easier to prepare releases,
+   and allows the author of each change to properly summarize it.
+4. **Update the documentation in `USAGE.md` if necessary**.
+   When writing prose, break lines [semantically](https://rhodesmill.org/brandon/2012/one-sentence-per-line/).
+   We don't have a strict maximum line length
+   (especially since things like long URLs can easily surpass them),
+   but you should start thinking about breaking lines once they're over approaching 100 characters in length.
+
+## PHP coding guidelines
+
+- Use a `\` before standard class names
```

Comment:
@Xymph this entry is just a placeholder for this section. Do you think there are specific recommendations we could add here? Or should we simply link to an existing PHP style guide or best practices document?

Reply:
For historical record, this came up in https://github.com/hamstar/Wikimate/pull/133#issuecomment-907209101.

Reply:
The main standards are the [PSRs](https://www.php-fig.org/psr/), and I've looked into some occasionally but am not well versed in any, and am not sure Wikimate even meets the basic coding standard in all relevant aspects.

The `\` specifies [global namespace](https://www.php.net/manual/en/language.namespaces.global.php), but as said, we don't (yet) use a namespace so I don't think it requires highlighting at this point.
I am simply not very experienced with namespaces, autoloading, composer, and similar modern technologies (although I worked extensively with Zend Framework 1, but that was during 2010-2014), so not in a strong position to recommend a style guide or best practices doc.

Reply:
Ok, let me remove this section for now, and maybe later on we can add one if we feel it's warranted.

Reply:
Done in the [latest push](https://github.com/hamstar/Wikimate/compare/2206bf4..f0cd8f9).